install_config/topics/ocp_osp_admin_tasks.adoc

```diff
@@ -30,6 +30,72 @@ $ openstack user create --password *<password> <username>*
 $ openstack role add --user <username> --project <project> _member_
 ----
 
+The default quotas assigned to new RH OSP projects are not high enough for
+{product-title} installations. Increase the quotas to at least 30 security
+groups, 200 security group rules, and 200 ports. If kuryr SDN is enabled, and
+particularly if namespace isolation is used, increase the quotas to at least
+100 security groups, 500 security group rules, and 500 ports.
+----
+$ openstack quota set --secgroups 30 --secgroup-rules 200 --ports 200 *<project>*
+----
+
+If Kuryr-SDN is used, with the namespace isolation feature enabled, there is
```

Comment:
the next steps can be removed too, as this is only for namespace isolation, and it is not supported in 3.10